cmd/mayactl/app/command/version.go

```diff
@@ -38,6 +41,7 @@ maya version
 	`,
 
 		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(checkLatestVersion(version.GetVersion()), util.Fatal)
 			fmt.Printf("Version: %s\n",
```

Comment:
let's move this to at the end of the execution hierarchy or run as go routine. @prateekpandey14 @AmitKumarDas @kmova what's your view on this ?

Reply:
@utkarshmani1997 I think its better to call at the last.